komodo/symlink/create_links.py

```diff
@@ -30,7 +44,15 @@ def _create_link(src, dst, link_dict):
         if existing_link == src:
             return
         os.remove(dst)
-        print(f"Existing symlink {dst} moved from {existing_link} to {src}")
+        implicitly_moved_sources = get_implicitly_moved_symlinks(dst, link_dict)
+        message = (
+            f"Existing symlink {dst} moved from {existing_link} to {src}."
+            + " Some symlinks were implicitly moved due to this."
```

Comment:
Looking better! I would move the punctuation dot to after the paranthesis. Or now that it is more explicit how it will look in Slack, maybe a colon and then just a comma-separated list without paranthesis.